### PR TITLE
Added support for QoS profiles

### PR DESCRIPTION
**Pull request by esteve**
Connects to ros2/ros2#66

### CODE REVIEW

**Comment by dirk-thomas** [comment]
+1

File: image_tools/src/cam2image.cpp [hunk 1]
```diff
@@ -58,7 +58,11 @@ int main(int argc, char * argv[])
 
   auto node = rclcpp::node::Node::make_shared("cam2image");
 
-  auto pub = node->create_publisher<sensor_interfaces::msg::Image>("image", 10);
+  rmw_qos_policies_t custom_qos_policies = rmw_qos_policies_default;
+  custom_qos_policies.depth = 10;
```
Comment (jacquelinekay) [inline image_tools/src/cam2image.cpp]: Will this number be configurable at some point (such as via command line args or a GUI tool)?

Comment (esteve) [inline image_tools/src/cam2image.cpp]: It should be, but I didn't want to make more changes than necessary, just update the code for the QoS changes.

**Comment by jacquelinekay** [comment]
+1

**Pull request merged**